llvm/lib/MC/MCParser/AsmLexer.cpp

```diff
@@ -878,7 +878,16 @@ AsmToken AsmLexer::LexToken() {
   case ' ':
   case '\t':
     IsAtStartOfStatement = OldIsAtStartOfStatement;
-    while (*CurPtr == ' ' || *CurPtr == '\t')
+#ifdef LLVM_DEBUG
```

Comment:
No need to use LLVM_DEBUG or write the lengthy comment about [] operator.

LLVM_USE_SANITIZER=Address builds would catch this issue with the new test case.